lib/firefox/webdriver/firefoxDelegate.js

```diff
@@ -38,7 +38,7 @@ class FirefoxDelegate {
       let bufferSize = this.firefoxConfig.geckoProfilerParams.bufferSize;
 
       const script = `Services.profiler.StartProfiler(${bufferSize},${interval},${featureString},
```

Comment:
I expect this to be `[${featureString}]` and `[${threadString}]` in order to be arrays, like what the new interface at https://searchfox.org/mozilla-central/rev/662de518b1686c4769320d6b8825ce4864c4eda0/tools/profiler/gecko/nsIProfiler.idl#37 expects.  Is there an automatic conversion happening?

Is it possible to figure out which version is correct at runtime?  'cuz this tooling might be targeting both old and new versions of Gecko.  In general this is hard -- it's the downside to using internal APIs in this way -- but if we can do it, we should try to do it.  Maybe we can figure it out from [`Function.length`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/length)?